packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -443,10 +443,26 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
     scroll(scrollValue);
   };
 
+  const getFirstVisibleTab = (tabs) => {
+    const containerSize = tabsRef.current[clientSize];
+    const containerLeftBound = Math.round(tabsRef.current.scrollLeft);
```

Comment:
Shouldn't we calculate the top & bottom bounds respectfully if the tabs are vertical? Would help if we add test for the vertical scenario too and maybe codesandbox to illustrate that it works.

Reply:
Changes are made. It will now work exactly the same in vertical orientation. I also added an extra testcase for this.